tests/test_random_param_search.py

```diff
@@ -11,7 +11,7 @@ def test_random_param_search(self):
         result = random_param_search(params_dict=params_dict, size=50)
 
         self.assertEqual(len(result), 5)
```

Comment:
since size=50 in line 11, we would want to assert len(result) to be equal to 50 as well